packages/core/src/Stores/contract-store.js

```diff
@@ -290,7 +288,6 @@ export default class ContractStore extends BaseStore {
                     this.accu_low_barrier || low_barrier,
                     null,
                     {
-                        color: is_dark_mode ? BARRIER_COLORS.DARK_GRAY : BARRIER_COLORS.GRAY,
```

Comment:
@akmal-deriv can we keep the functionality for changing the main color of the barrier from deriv-app? in case the color will be reconsidered in future?

Reply:
I didn't remove this functionality as mentioned, this line just changes colors from grey to dark-grey for dark theme, which is not needed anymore.